### ISSUE TITLE
commentedOutCode does not detect commented out assignments

### ISSUE THREAD

**Issue by rittneje** [go-critic/go-critic#1244]
None of these commented out lines are flagged.

```go
// var x = 123

// const y = 456

func Foo() {
	// z := 789
}
```

**Comment by peakle** [comment]
commentedOutCode rule checks only body of the function, so comments outside the function will not trigger this checker.
And if comment length too short (<15 letters) it won't be activated either.

**Comment by quasilyte** [comment]
The 15 letters limitation was added to avoid false positives.
If we can improve the accuracy and detect more real commented-out code without adding new false positives, it would be a good change. For instance, seeing `:=` inside the comment could be a strong point towards "it's a Go code". Weighted system could help to make decisions when some of the thresholds are just too strict (like min of 15 chars).

**Comment by cristaloleg** [comment]
We can add a parameter to specify min length for more conservative users, should be a trivial thing.

**Comment by hendrywiranto** [comment]
just want to mention that this issue should have been closed when the PR is merged @cristaloleg

**Comment by cristaloleg** [comment]
Nice catch, thanks!
